packages/paste-core/layout/media-object/src/MediaFigure.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import type {BoxProps} from '@twilio-paste/box';
+import {isMarginTokenProp} from '@twilio-paste/style-props';
+import type {Space} from '@twilio-paste/style-props';
+
+export interface MediaFigureProps extends React.HTMLAttributes<any>, Pick<BoxProps, 'as' | 'element' | 'variant'> {
+  align?: 'start' | 'end';
+  children: NonNullable<React.ReactNode>;
+  spacing?: Space;
+}
+
+const MediaFigure = React.forwardRef<HTMLElement, MediaFigureProps>(
+  ({align, as = 'span', children, element = 'MEDIA_FIGURE', spacing, ...props}, ref) => {
+    return (
+      <Box
+        element={element}
+        {...safelySpreadBoxProps(props)}
+        as={as}
+        display="flex"
+        flexShrink={0}
+        marginLeft={align === 'end' ? spacing : undefined}
+        marginRight={align === 'start' ? spacing : undefined}
+        ref={ref}
+      >
+        {children}
+      </Box>
+    );
+  }
+);
+
+MediaFigure.displayName = 'MediaFigure';
+
+MediaFigure.defaultProps = {
```

Comment:
Just curious why you opted to using defaultProps here in the standard/classic style, vs setting it in the destructuring on line 15 along with `as` and `element`

Reply:
Good catch. I'll refactor those.